tests/migration/migration.go

```diff
@@ -3141,7 +3129,18 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 
 			By("starting a VMI on the first node of the list")
```

Comment:
Nit: I think you can probably combine this comment with the following comment since this is now only referring to labeling the node. Maybe something like this? 

```go
    By("starting a migratable VMI with 2 dedicated CPU cores on the first node of the list")
    libnode.AddLabelToNode(nodes[0].Name, testLabel1, "true")
    vmi := libvmi.NewAlpine(
```

Reply:
I do not think anything changed in the test logic, so I see no reason to change the logging here.

I usually do not like these comments at all, but some find them useful when reading the test logs (when it fails) to recognize better at what point the failure occurred. Adding the mentioned details will not really add anything valuable to someone that reads the logs, they will still need to visit the code to understand what failed.